style.css

```diff
@@ -1,825 +1,810 @@
+.header__inner-menu-btn::after {
+  content: '';
+  position: absolute;
+  width: 26.4px;
+  height: 2.61px;
+  background-color: #000;
+  top: 29.66px;
+  left: 21.49px;
+  box-shadow: 0 7.08px;
 }
 
-.header__menu-item {
-  height: 3.48px;
-  width: 34.7px;
-  background: black;
-  margin-bottom: 5.96px;
+.head {
+	background: url(img/bg-head.png) no-repeat;
+	background-position: center;
+	background-size: cover;
+	min-height: 522px;
 }
 
-.header__language {
-  position: absolute;
-  left: 27%;
-  top: 42px;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 20px;
-  color: #fff;
-  z-index: 2;
+.head__img-title {
+  font-size: 50px;
+  line-height: 1;
+  padding-left: 26px;
+  padding-top: 365px;
+  letter-spacing: 1px;
 }
 
-.header__vector {
-  position: absolute;
-  left: 29.5%;
-  top: 54px;
-  z-index: 2;
+.head__img-vector {
+  display: block;
+  margin: 23px auto;
 }
 
-.first {
-  width: 100%;
-  display: grid;
-  grid-template-columns: 32% 68%;
-  grid-template-rows: 900px;
+.second {
+	min-width: 320px;
+  height: 601px;
+  background: #fae8e4;
 }
 
-.first__sidebar {
-  grid-column: 1;
-  background: black;
+.second__bag {
+  display: block;
+  margin: 0 auto;
+  height: 230px;
+  width: 218px;
+  padding-top: 50px;
+  margin-bottom: 39px;
 }
 
-.first__sidebar-social {
-  display: flex;
-  justify-content: space-between;
-  width: 155px;
-  padding: 816px 51px 0;
+.second__box {
+	width: 320px;
+  height: 281px;
+  background: #fff;
+  margin: 0 auto;
 }
 
-.collection {
-  grid-column: 2;
-  background: yellow;
+.second__container {
+	width: 254px;
+	height: 226px;
+  margin: 0 auto;
 }
 
-.step {
-  position: absolute;
-  width: 100%;
-  display: grid;
-  grid-template-columns: 549px 549px;
-  grid-template-rows: 600px;
-  justify-content: center;
-  top: 168px;
-  z-index: 2;
-  grid-column-gap: 68px;
+.second__title {
+	font-size: 20px;
+  line-height: 1;
+  padding-top: 18px;
+  letter-spacing: .7px;
+  margin-bottom: 4px;
 }
 
-.img__img {
-  height: 562px;
+.second__subtitle {
+	font-size: 10px;
+  line-height: 1;
+  letter-spacing: 1px;
+  opacity: .5;
+  margin-top: 0;
+  margin-bottom: 14px;
 }
 
-.text {
-  height: 600px;
+.second__text {
+	font-size: 9px;
+  line-height: 1.44;
+  opacity: .5;
+  margin-bottom: 23px;
 }
 
-.text__container {
-  padding: 50px 0;
-  letter-spacing: 2px;
+.second__block-color {
+	display: flex;
 }
 
-.text__title {
-  width: 549px;
-  font-style: normal;
-  font-weight: bold;
-  font-size: 80px;
-  line-height: 82px;
-  margin-bottom: 0;
+.second__block-color-gray,
+.second__block-color-pink,
+.second__block-color-blue,
+.second__block-color-green {
+	width: 23px;
+	height: 23px;
+	border-radius: 50%;
 }
 
-.text__subtitle {
-  font-family: Montserrat, sans-serif;
-  font-style: normal;
-  font-weight: normal;
-  font-size: 15px;
-  line-height: 21px;
-  letter-spacing: .5px;
-  margin-bottom: 52px;
+.second__block-color-gray {
+	background-color: #dedede;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)